Assets/BossRoom/Scripts/Client/Game/State/ClientCharSelectState.cs

```diff
@@ -403,7 +403,7 @@ public void OnPlayerExit()
             // first disconnect then return to menu
             var gameNetPortal = GameObject.FindGameObjectWithTag("GameNetPortal").GetComponent<GameNetPortal>();
             gameNetPortal.RequestDisconnect();
-            SceneManager.LoadScene("MainMenu");
+            SceneLoader.Instance.LoadScene("MainMenu", LoadSceneMode.Single);
```

Comment:
instead of adding single everywhere, couldn't we add a default value to SceneLoader's LoadScene?

Reply:
We could, but I like having it that way, like how it is with NetworkSceneManager. It makes it more deliberate to chose to load a scene in Single mode.

Reply:
And the static LoadScene method has default parameters, just like Unity's SceneManager does too https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadScene.html
This way, users that want simple calls can just use the default behaviour, without preventing them to be explicit about it (those that want to be explicit can still "override" the default param with the same default param)

Reply:
I don't see any static LoadScene method with default parameters in NetworkSceneManager, only in Unity's SceneManager, am I looking in the wrong place? Either way, I'll add it some to this new method then